src/sagas/contacts/sagas.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import {
+  call,
+  put,
+} from 'redux-saga/effects';
+import ContactsService from '../../services/contacts';
+import {
+  startContactsFetch,
+  contactsFetchFailed,
+  contactsUpdated,
+} from '../../actions/contacts';
+import { getProfile } from '../chat/sagas';
+import type { AddContactAction } from '../../actions/contacts';
+
+/**
+ * @desc Fetch list of contacts.
+ * @return {void}
+ */
+export function* fetchContacts(): Generator<*, *, *> {
+  try {
+    yield put(startContactsFetch());
```

Comment:
You shouldn't put that action there. It would cause an infinite loop (because `fetchContacts` saga is called whenever `START_CONTACTS_FETCH` action is dispatched).

Reply:
Done